### PR TITLE
Fallback to old field-caps for old remote clusters

### PR DESCRIPTION
**Pull request by dnhatn**
This change falls back to the old field-caps action if the remote cluster has not been updated to 8.16 or later.

### CODE REVIEW

File: x-pack/plugin/esql/qa/server/multi-clusters/build.gradle [hunk 1]
```diff
@@ -18,8 +18,8 @@ dependencies {
 }
 
 def supportedVersion = bwcVersion -> {
-  // ESQL requires its own resolve_fields API
-  return bwcVersion.onOrAfter(Version.fromString("8.16.0"));
+  // CCS in ES|QL available in 8.13
+  return bwcVersion.onOrAfter(Version.fromString("8.13.0"));
```
Comment (nik9000) [inline x-pack/plugin/esql/qa/server/multi-clusters/build.gradle:22]: :+1:

**Review by nik9000 — COMMENTED** [review]
Seems right to me. Surprisingly not that bad to keep this rolling during the rolling upgrade.

**Comment by dnhatn** [comment]
```
./gradlew ":x-pack:plugin:ml:qa:ml-with-security:yamlRestTest" --tests "org.elasticsearch.smoketest.MlWithSecurityIT.test {yaml=ml/calendar_crud/Test deleted job is removed from calendar}" -Dtests.seed=3DABDEA90286336D -Dtests.locale=kw-GB -Dtests.timezone=America/Porto_Acre -Druntime.java=22
```

**Review by n1v0lg — APPROVED** [review]

**Comment by dnhatn** [comment]
@nik9000 @n1v0lg Thanks for reviewing.

**Pull request merged**